web/admin/application/library/IvozProvider/Klear/Ghost/Companies.php

```diff
@@ -118,14 +119,23 @@ public function getCurrentDayMaxUsage(CompanyDto $companyDto)
                 $companyDto->getId()
             );
 
+            $dbAmount = $companyDto->getMaxDailyUsage();
+            if ($dbAmount == AbstractBalanceService::UNLIMITED_MAX_DAILY_USAGE) {
+                $dbAmount = '∞';
+            }
+
             $currencySymbol = $dataGateway->remoteProcedureCall(
                 Company::class,
                 $companyDto->getId(),
                 'getCurrencySymbol',
                 []
             );
 
-            return sprintf("%s %s", $amount, $currencySymbol);
+            if ($dbAmount == $amount) {
+                return sprintf("%s %s", $amount, $currencySymbol);
+            } else {
+                return sprintf("<span style='color:red;'>%s %s</span>", $amount, $currencySymbol);
```

Comment:
I would add a `title="Amount differs from one stored"` in this span, so users can hover and get some info why the text is red.